samwise.js

```diff
@@ -1,23 +1,21 @@
 /**
```

Comment:
SCRIPT-Consider using more descriptive naming for this boolean variable to clarify its purpose.